pipeline/src/transformers/article.ts

```diff
@@ -82,9 +84,9 @@ function transformLabelType(
 }
 
 export const transformArticle = (
-  document: ArticlePrismicDocument
+  document: ArticlePrismicDocument,
 ): ElasticsearchArticle => {
-  const { data, id, first_publication_date } = document;
+  const { data, id, first_publication_date: firstPublicationDate } = document;
```

Comment:
Camel case gets flagged, but since we need this (it's how it is in Prismic), naming it something else calms the alert down.